packages/admin-ui/ui/src/domain/settings/regions/components/shipping-option-card/edit-modal.tsx

```diff
@@ -72,7 +79,7 @@ const EditModal = ({ open, onClose, option }: Props) => {
             <div>
               <p className="inter-base-semibold">Fulfillment Method</p>
               <p className="inter-base-regular text-grey-50">
-                {option.data.id} via {option.provider_id}
+                {option.data.id as string} via {option.provider_id}
```

Comment:
Not necessary in terms of functionality, but since shipping_option.data is typed as `Record<string, unknown>` the `id` is unknown which could be a Component, so TS complains that we are providing multiple children to a P tag which isn't allowed.